README.md

```diff
@@ -60,6 +60,7 @@ $ bower install validator-js
 - **isBefore(str [, date])** - check if the string is a date that's before the specified date.
 - **isIn(str, values)** - check if the string is in a array of allowed values.
 - **isCreditCard(str)** - check if the string is a credit card.
+- **isISIN(str)** - check if the string is an ISIN.
```

Comment:
Can you link to https://en.wikipedia.org/wiki/International_Securities_Identification_Number here?